packages/combine-adapters/src/adapter/adapter.ts

```diff
@@ -111,7 +110,9 @@ class CombineAdapters implements TCombinedAdapterInterface {
     }
 
     const handleStatusChange = (nextStatus: TAdapterStatusChange['status']) => {
-      if (this.#getIsAdapterUnsubscribed()) return;
+      if (this.#getIsAdapterUnsubscribed()) {
+        return;
+      }
```

Comment:
This repeats a lot, biome doesn't like return without curely braces (readability)